src/migrateUIFolder.ts

```diff
@@ -195,6 +197,11 @@ const accumulateStructure = ({
 export function migrateUIFolder(
   legacyUIFolder: ContentRecord,
   servers: { [serverKey: string]: { dataModel: DataModel; url: string } },
+  calculatedMeasures: {
```

Comment:
Let's make this optional and put it at the last option to avoid breaking changes.